static/js/publisher/form/AccordionHelp.js

```diff
@@ -0,0 +1,56 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+class AccordionHelp extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggleHelp = this.toggleHelp.bind(this);
+
+    this.state = {
+      open: false
+    };
+  }
+
+  toggleHelp() {
+    const { open } = this.state;
+
+    this.setState({
+      open: !open
+    });
+  }
+
+  render() {
+    const { open } = this.state;
+    const { name, children } = this.props;
+
+    const classNames = ["row"];
+    let verb = "Hide";
+    if (!open) {
+      classNames.push("u-hide");
+      verb = "Show";
+    }
+
+    return (
+      <Fragment>
+        <p className={`p-form-help-text ${open ? "u-no-margin--bottom" : ""}`}>
+          <a role="button" onClick={this.toggleHelp}>
+            {verb} {name.toLowerCase()}
```

Comment:
That's very specific "verb+name" approach. I guess having just 2 props for 'show label' and 'hide label' would be  a bit more generic and reusable. But it can be changed later if needed elsewhere.